sdk/cognitiveservices/azopenai/client_shared_test.go

```diff
@@ -74,10 +72,10 @@ func init() {
 		chatCompletionsModelDeployment = "gpt-4"
 		openAIChatCompletionsModel = "gpt-4"
 	} else {
-		if err := godotenv.Load(); err != nil {
-			fmt.Printf("Failed to load .env file: %s\n", err)
-			os.Exit(1)
-		}
+		// if err := godotenv.Load(); err != nil {
```

Comment:
Do you want to keep this code?

Reply:
Just did a little update - I needed it and incorrectly thought I didn't :)